example_sns_init.yaml

```
@@ -25,96 +31,338 @@ description: >
+    #
+    # If this value is set to 1 day, then a change in the majority vote at the
+    # end of a proposal's original voting period results in an extension of the
+    # voting period by an additional day. Another change at the end of the
+    # extended period will cause the voting period to be extended by another
+    # half-day, etc.
+    #
+    # The total extension to the voting period will never be more than twice
+    # this value.
+    #
+    # For more information, please refer to
+    # https://wiki.internetcomputer.org/wiki/Network_Nervous_System#Proposal_decision_and_wait-for-quiet
     maximum_wait_for_quiet_deadline_extension: 1 day
 
-# It is fine to leave these alone.
+# Configuration of SNS neuron-related fields. These fields have no effect on the
+# NNS proposal, but will affect the SNS itself.
 Neurons:
-    minimum_creation_stake: 10 tokens
+    # The minimum amount of SNS tokens an SNS ledger account must have to stake
+    # a neuron. The larger the minimum, the more tokens must be acquired to be
+    # able to vote.
+    minimum_creation_stake: 1 tokens
 
-# It is fine to leave these alone.
+# Configuration of voting-related fields of the SNS. These fields have no effect
+# on the NNS proposal created by this file, but will affect the SNS itself.
 Voting:
-    minimum_dissolve_delay: 26 weeks
+    # The minimum dissolve delay a neuron must have to be able to cast votes on
+    # proposals. The higher the minimum, the longer users will need to be
+    # aligned with the SNS to vote. However, the higher the minimum, the longer
+    # users will not be able to access the SNS tokens used to stake the neuron.
+    minimum_dissolve_delay: 1 month
 
+    # Configuration of the various voting power bonuses that are applied to
+    # neurons to incentivize alignment with the best interest of the DAO.
     MaximumVotingPowerBonuses:
+        # Users with a higher dissolve delay are incentivized to take the
+        # long-term interests of the SNS into consideration when voting. To
+        # reward this long-term commitment, this bonus can be set to a
+        # percentage greater than zero, which will result in neurons having
+        # their voting power increased in proportion to their dissolve delay.
+        #
+        # If you do not want this bonus to be applied for neurons with higher
+        # dissolve delay, set `bonus` to `0%` and those neurons will not receive
+        # higher voting power.
         DissolveDelay:
+            # This parameter sets the maximum dissolve delay a neuron can have.
+            # When reached, the maximum dissolve delay bonus will be applied.
             duration: 8 years
+            # If a neuron's dissolve delay is `duration`, its voting power will
+            # be multiplied by this `bonus` amount.
             bonus: 100%
 
+        # Users with neurons staked in the non-dissolving state for a long
+        # period of time are incentivized to take the long-term interests of
```

```suggestion
        # Users with tokens staked in neurons for a long
        # period of time are incentivized to take the long-term interests of
```
I think this applies for non-dissolving and dissolving neurons